proxy/scripts/test_upgrade.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+DEPLOYED_TAG=$(cat $GITHUB_WORKSPACE/proxy/DEPLOYED)
+DEPLOYED_VERSION=$(echo $DEPLOYED_TAG | cut -d '-' -f 1)
+DEPLOYED_DIR=$GITHUB_WORKSPACE/deployed-proxy/
+
+# TODO: remove when upgrade proxy with using new node js
+# or install old version of node
+if [[ $NODE_VERSION != 12.* ]]
+then
+    echo "Skip upgrade check because of incompatible node.js version"
+    exit 0
+fi
+
+git clone --branch $DEPLOYED_TAG https://github.com/$GITHUB_REPOSITORY.git $DEPLOYED_DIR
+
+npx ganache-cli --gasLimit 8000000 --quiet &
+GANACHE_PID=$!
+
+cd $DEPLOYED_DIR
+yarn install || exit $?
+cd proxy
+CHAIN_NAME_SCHAIN="Test" PRODUCTION=true VERSION=$DEPLOYED_VERSION npx hardhat run migrations/deploySkaleManagerComponents.ts --network localhost || exit $?
```

Comment:
Why do we need variable "PRODUCTION" here?

Reply:
Ooops, removed